src/celengine/timelinephase.h

```diff
@@ -42,7 +41,7 @@ class TimelinePhase
         return m_endTime;
     }
 
-    ReferenceFrame* orbitFrame() const
+    const std::shared_ptr<const ReferenceFrame>& orbitFrame() const
```

Comment:
why `const` here?

Reply:
I prefer to declare references as const whenever possible to avoid unneeded-modification-bugs.

Reply:
Celestia already has a bit more than allowed `const_cast`s :)

Reply:
I didn't see any yet. I may start removing it in another PR.

Reply:
It's better not to introduce new

Reply:
I dont, do I?

Reply:
Potentially with `const` put here and there once we will face a situation when api change or static_const is required.

Reply:
Still feel const is better. If someone really need to modify particular data, then maybe it's a time to rethink general approach. Personally, I like to know exactly where and who modify particular data.